app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -47,6 +51,12 @@ class MainActivity : AppCompatActivity() {
             Toast.makeText(this, quizViewModel.checkAnswer(false), Toast.LENGTH_SHORT).show()
         }
 
+        cheatButton.setOnClickListener {
+            val answerIsTrue = quizViewModel.currentQuestionAnswer
```

Comment:
I would rename this val to `answerQuestion` or `currentAnswerQuestion` to keep consistency with `quizViewModel.currentQuestionAnswer`